src/components/Text/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './Text';
+export {TYPOGRAPHY_VARIANTS, text} from './text/text';
+export {TEXT_COLORS, colorText} from './colorText/colorText';
```

Comment:
Let's name "not a component or hook or class" files in dash-case.

Reply:
`colorText` is a component ("block" in BEM paradigm). But yes it's not a "react" component

Reply:
and it's better the entity name and the file name match